core/src/main/resources/reference.conf

```diff
@@ -550,6 +566,183 @@ datastax-java-driver {
+        # this can happen.
+        // errors.aborted_requests,
+
+        # The number of times this node replied with a WRITE_TIMEOUT error (exposed as a Counter).
+        #
+        # Whether this error is rethrown directly to the client, rethrown or ignored is determined
+        # by the RetryPolicy.
+        // errors.write_timeouts,
+
+        # The number of times this node replied with a READ_TIMEOUT error (exposed as a Counter).
+        #
+        # Whether this error is rethrown directly to the client, rethrown or ignored is determined
+        # by the RetryPolicy.
+        // errors.read_timeouts,
+
+        # The number of times this node replied with an UNAVAILABLE error (exposed as a Counter).
+        #
+        # Whether this error is rethrown directly to the client, rethrown or ignored is determined
+        # by the RetryPolicy.
+        // errors.unavailables,
+
+        # The number of times this node replied with an error that doesn't fall under other errors.*
+        # metrics (exposed as a Counter).
+        // errors.others,
+
+        # The total number of errors on this node that caused the RetryPolicy to trigger a retry
+        # (exposed as a Counter).
+        #
+        # This is a sum of all the other retries.* metrics.
+        // retries.total,
+
+        # The number of errors on this node that caused the RetryPolicy to trigger a retry, broken
+        # down by error type (exposed as Counters).
+        // retries.aborted,
+        // retries.read_timeout,
+        // retries.write_timeout,
+        // retries.unavailable,
+        // retries.other,
+
+        # The total number of errors on this node that were ignored by the RetryPolicy (exposed as a
+        # Counter).
+        #
+        # This is a sum of all the other ignores.* metrics.
+        // ignores.total,
+
+        # The number of errors on this node that were ignored by the RetryPolicy, broken down by
+        # error type (exposed as Counters).
+        // ignores.aborted,
+        // ignores.read_timeout,
+        // ignores.write_timeout,
+        // ignores.unavailable,
+        // ignores.other,
+
+        # The number of speculative executions triggered by a slow response from this node (exposed
+        # as a Counter).
+        // speculative_executions,
+
+        # The number of errors encountered while trying to establish a connection to this node
+        # (exposed as a Counter).
+        #
+        # Connection errors are not a fatal issue for the driver, failed connections will be retried
+        # periodically according to the reconnection policy. You can choose whether or not to log
+        # those errors at WARN level with the connection.warn-on-init-error option.
+        #
+        # Authentication errors are not included in this counter, they are tracked separately in
+        # authentication_errors.
+        // connection_errors,
```

Comment:
Should `connection_errors` and `authentication_errors` be under `errors.`?

Reply:
They relate to connections while the others are more specifically for requests, so maybe I should have two sub-categories.

Reply:
good point, maybe differentiating into `connection_errors` and `request_errors` or something, and then renaming the current `connection_errors` to something like more specific like `startup_errors` or something like that.